packages/main/src/themes/Button.css

```diff
@@ -205,10 +217,17 @@ bdi {
 	font-weight: var(--_ui5_button_emphasized_font_weight);
 }
 
-:host([design="Emphasized"]:not([active]):not([non-interactive]):not([_is-touch]):hover) {
+:host([design="Emphasized"]:not([active]):not([non-interactive]):not([_is-touch]):hover),
+:host([design="Emphasized"]:not([active]):not([non-interactive]):not([_is-touch]).ui5_hovered) {
 	background-color: var(--sapButton_Emphasized_Hover_Background);
 	border-color: var(--sapButton_Emphasized_Hover_BorderColor);
 }
+
+/* :host([design="Emphasized"]:not([active]):not([non-interactive]):not([_is-touch]).ui5_hovered){
```

Comment:
commented code?

Reply:
Missed when refactoring. It's removed now. Thanks